packages/callbacks/example/battery/lib/main.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'dart:ffi';
+import 'package:tizen_interop/6.5/tizen.dart';
+import 'package:tizen_interop_callbacks/tizen_interop_callbacks.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+MyAppState? myAppState;
```

Comment:
This may be declared as a static member of the `MyAppState` class.

The code still may not work as expected if multiple instances of `MyAppState` are created. Do you have a more generalized solution? (e.g. somehow utilize the `userData` parameter)

Reply:
solution - not really, the callback can only be static/free function with only access to static/globals, if we allowed userData to be any object and store the object internally that would probably be easier for users, but as it is the user would have to map values in userData to some context information himself/herself.

Reply:
It would be the best if a Dart object (a `MyAppState` instance in this case) can be represented as a native pointer and the pointer can be converted back to a Dart object reference. As far as I know that's not allowed in Dart at the moment so they should be explicitly mapped and managed in user code (using a `Map` or something). How about adding such mapping (a private static `Map`) to the example so that `MyAppState` becomes more self-contained and reusable in that sense?